src/Avalonia.Controls/ComboBox.cs

```diff
@@ -290,26 +290,12 @@ protected override void OnPointerWheelChanged(PointerWheelEventArgs e)
         protected override void OnPointerPressed(PointerPressedEventArgs e)
         {
             base.OnPointerPressed(e);
-            if(!e.Handled && e.Source is Visual source)
+            if(e.Handled)
             {
-                if (_popup?.IsInsidePopup(source) == true)
-                {
-                    e.Handled = true;
-                    return;
-                }
-            }
-
-            if (IsDropDownOpen)
-            {
-                // When a drop-down is open with OverlayDismissEventPassThrough enabled and the control
-                // is pressed, close the drop-down
-                SetCurrentValue(IsDropDownOpenProperty, false);
-                e.Handled = true;
-            }
-            else
-            {
-                PseudoClasses.Set(pcPressed, true);
+                return;
             }
+            e.Handled = true;
+            PseudoClasses.Set(pcPressed, true);
```

Comment:
Setting the pressed state makes the `ComboBox` appear pressed whenever an item is being selected inside the popup, this isn't right.